src/components/Answers.module.scss

```diff
@@ -44,8 +44,30 @@
   align-items: center;
   width: 100%;
   padding: 15px;
+  flex-wrap: wrap;
+
+  > *:first-child {
+    margin-right: 10px;
+  }
 
   * {
     margin: 0px;
   }
+
+  .correctHint {
+    display: none;
+    @include variables.screen-size-sm-and-up {
+      display: block;
+    }
+  }
+
+  // Ensure code elements don't overflow
```

Comment:
Just for consistency: you sometimes put the comment above the class name and sometimes on the rule declaration. Makes more sense on the declaration, right? As in `Question.module.scss`

Reply:
Makes sense, I'll move it.